make.js

```diff
@@ -21,7 +30,7 @@ setTimeout(function() {
     (function(t, oldTarget){
 
       // Wrap it
-      global.target[t] = function(force) {
+      global.target[t] = function(localArgs, force) {
```

Comment:
I frankly can't remember why I put `force` in here.. it's not even documented :) maybe remove it?